public/index.html

```diff
@@ -8,6 +8,7 @@
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Web site created using create-react-app" />
   <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+  <script src="https://kit.fontawesome.com/101c647319.js" crossorigin="anonymous"></script>
```

Comment:
we are already using react-icons. please use that.

Reply:
> we are already using react-icons. please use that.

(mistakenly clicked on resolve conversation) anyway added react-icon have a look @akshitarora921